frontend/src/store/playbackManager.ts

```diff
@@ -474,158 +476,93 @@ class PlaybackManagerStore {
 
   private get _pendingProgressReport(): boolean {
     return (
+      !this._isProgressUpdating &&
       reactiveDate.value.valueOf() - this._state.lastProgressUpdate >=
         progressReportInterval &&
       this.status !== PlaybackStatus.Stopped &&
       this.status !== PlaybackStatus.Error
     );
   }
 
+  private get _mediaSessionMetadata(): MediaMetadata | null {
+    return this.currentItem && window.navigator.mediaSession
```

Comment:
@ThibaultNocchi I moved it to the watchEffect and changed the method to put a raw object instead of a `MediaMetadata` instance. I also used the `unknown` translation key for missing fields (or shall we leave those blank and use '' instead?)